### PR TITLE
Complete the project

### PR DESCRIPTION
**Pull request by hopeforkoami**
in this branch i complete the todo list project

### CODE REVIEW

File: src/modules/task.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+
+  getCompleted() {
+    return this.completed;
+  }
+
+  setCompleted(statut) {
```
Comment (Camilovelag) [inline src/modules/task.js:12]: Correct the word `statut` spelling to keep clear the meaning of the function's parameter.

File: src/modules/tasklist.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+  generateTodoIndex() {
+    return this.localData.length + 1;
+  }
+
+  refreshIndex() {
+    this.localData.forEach((task, index) => {
+      task.index = index + 1;
+    });
+    console.log('end of the refresh');
```
Comment (Camilovelag) [inline src/modules/tasklist.js:23]: Delete this `console.log`

File: src/modules/tasklist.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import Task from './task.js';
+
+export default class TaskList {
+  constructor() {
+    this.localData = [];
+    const tempsData = JSON.parse(localStorage.getItem('taskList')) || [];
+    if (tempsData.length > 0) {
+      tempsData.forEach((element) => {
+        console.log(element);
```
Comment (Camilovelag) [inline src/modules/tasklist.js:9]: Delete this `console.log`

File: src/modules/tasklist.js (hunk 2 above)
Comment (Camilovelag) [inline src/modules/tasklist.js:23]: Delete this `console.log`

**Review by Camilovelag — COMMENTED** [review]
# Hello, @hopeforkoami 👋

Good job so far!
Your project is complete and looks great, but in order to improve the JS's good practices, I'd like to suggest some changes.

done

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Setup webpack to insert images_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not mandatory; these are just suggestions according to my point of view._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Review by Camilovelag — APPROVED** [review]
# Hello @hopeforkoami 👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Congratulations

**Comment by hopeforkoami** [comment]
thank you 🤝 🤝

**Pull request merged**